### ISSUE TITLE
Uploadable with doctrine inheritance not working

### ISSUE THREAD

**Issue by JarekW** [fsi-open/doctrine-extensions#39]
example to reproduce:

``` php
/**
 * @Entity
 * @InheritanceType("SINGLE_TABLE")
 */
class Person
{
    // ...
}
```

``` php
/**
 * @Entity
 */
class Employee extends Person
{
    /**
     * @ORM\Column(type="string", length=255, nullable=true)
     * @FSi\Uploadable(targetField="file")
     */
    private $filePath;

    private $file;
}
```

Exception:

```
Mapping "Uploadable" in property "filePath" of class "Person" has 
"targetField" set to "file", which doesn't exist.
```

**Comment by szymach** [comment]
#79 suggests this is resolved

**Comment by szymach** [comment]
Closed by #79
